srcs/HandlingString/HandlingString.cpp

```diff
@@ -0,0 +1,325 @@
+}
+
+std::string HandlingString::skipping_emptyword(std::string const &word)
+{
+	std::istringstream	splited_words(word);
+	std::string			src;
+	std::string			all_src;
+
+	while (splited_words >> src)
+		all_src = all_src + src;
+	return (all_src);
+}
+
+bool	HandlingString::check_under_intmax(std::string const &word)
+{
+	size_t	pos = 0;
+	size_t	sum = 0;
+
+	while (word[pos] != '\0')
+	{
+		if (std::isdigit(word[pos]) == false && word[pos] != ';')
+			return (false);
+		pos++;
+	}
+	if (pos > 12)  // INT_MAXの桁数を明らかに超えるなら計算の必要はない
+		return (false);
+	pos = 0;
+	while (word[pos] != '\0')
+	{
+		sum = sum * 10 + (word[pos] - '0');
+		if (sum >= INT_MAX)
+			return (false);
+		pos++;
+	}
+	return (true);
+}
+
+int	HandlingString::str_to_int(std::string const &word)
+{
+	size_t	pos = 0;
+	size_t	sum = 0;
+
+	pos = 0;
+	while (word[pos] != '\0')
+	{
+		sum = sum * 10 + (word[pos] - '0');
+		pos++;
+	}
+	return (sum);
+}
+
+double HandlingString::str_to_double(std::string word)
+{
+	std::istringstream iss(word);
+    double result;
+    iss >> result;
+    return result;
+}
+
+bool	HandlingString::check_lastword_semicoron(std::string const &word)
+{
+	size_t	pos = 0;
+	size_t	semicoron_count = 0;
+
+	while (word[pos] != '\0')
+	{
+		if (word[pos] == ';')
+			semicoron_count++;
+		pos++;
+	}
+	if (semicoron_count != 1)
+		return (false);
+	if (word[pos - 1] != ';')
+		return (false);
+	return (true);
+}
+
+std::string	HandlingString::skip_lastsemicoron(std::string const &word)
+{
+	size_t		pos = 0;
+	std::string	return_str;
+	size_t		index = word.find(';');
+
+	if (index == std::string::npos)
+		return (word);
+	while (word[pos] != ';')
+	{
+		return_str = return_str + word[pos];
+		pos++;
+	}
+	return (return_str);
+}
+
+void	HandlingString::error_show(std::string const &word, size_t const &pos)
+{
+	std::string error_one = "<< missed word! >>";
+	std::string error_two = "<< missed line >>";
+	std::string error_three = "===============";
+
+	std::cout << error_three << std::endl;
+	std::cout << error_one << std::endl;
+	std::cout << "* " << word << std::endl;
+	std::cout << error_two << std::endl;
+	std::cout << "line > " << pos << std::endl;
+	std::cout << error_three << std::endl;
+}
+
+bool	HandlingString::compare_word(std::string const &tgt_string, std::string const &key)
+{
+	if (key == tgt_string)
+		return (true);
+	return (false);
+}
+
+std::string HandlingString::obtain_second_word(std::string const &line)
+{
+	std::istringstream	splited_woeds(line);
+	std::string			f_word;
+	std::string			s_word;
+	std::string			t_word;
+
+	splited_woeds >> f_word >> s_word >> t_word;
+	return (s_word);
+}
+
+std::string HandlingString::int_to_str(size_t pos)
+{
+	std::string result;
+
+    if (pos == 0)
+	{
+        result = "0";
+		return result;
+	}
+	while (pos > 0)
+	{
+		result =  static_cast<char>('0' + pos % 10) + result;
+		pos /= 10;
+	}
+    return result;
+}
+
+void HandlingString::show_vector_contents(std::vector<std::string> subject)
```

Comment:
```suggestion
void HandlingString::show_vector_contents(const std::vector<std::string> &subject)
```

IMO:
const refで受け取れそうですね
（iteratorをconst_iteratorへの変更が必要になりますが...）

Reply:
これ使っていないので消しました